LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -5607,5 +5610,328 @@ LocLongDescription="Free <Ability:WeaponName/> attacks on any enemies that enter
+LocFlyOverText="Ruthless"
+LocPromotionPopupText="<Bullet/> By cleverly combining the effects of Brutality and Ruthless, you can kill multiple enemies per turn with your Sawed-Off Shotgun.<br/><Bullet/> There is no limit to the number of activations per turn.<br/>"
+
+[LW_ZoneOfControl X2AbilityTemplate]
+LocFriendlyName = "Zone of Control"
+LocHelpText = "All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties of <Ability:ZONE_CONTROL_AIM_PENALTY/> Aim and <Ability:ZONE_CONTROL_MOBILITY_PENALTY/> Mobility."
+LocLongDescription = "All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties to Aim and Mobility."
+LocPromotionPopupText = "<Bullet/> All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties of <Ability:ZONE_CONTROL_AIM_PENALTY/> Aim and <Ability:ZONE_CONTROL_MOBILITY_PENALTY/ Mobility.<br/>"
+
+[LW_ApexPredator X2AbilityTemplate]
+LocFriendlyName = "Apex Predator"
+LocFlyOverText = "Apex Predator"
+LocHelpText = "Critical Hits have a chance to panic the target and nearby enemies."
+LocLongDescription = "Critical Hits have a chance to panic the target and nearby enemies."
+LocPromotionPopupText = "<Bullet/>"Critical Hits have a chance to panic the target and nearby enemies.<br/>"
+
+[LW_BlindingProtocol X2AbilityTemplate]
+LocFriendlyName = "Blinding Protocol"
+LocFlyOverText = "Blinding Protocol"
+LocHelpText = "Send the Gremlin to deploy a small flashbang on an enemy target, disorienting all enemy units within a small radius (reducing their Aim by <Ability:DISORIENTED_AIM_ADJUST/>and Mobility by <Ability:DISORIENTED_MOBILITY_ADJUST/>"
+LocLongDescription = Send the Gremlin to deploy a small flashbang on an enemy target, disorienting all enemy units within a small radius (reducing their Aim by <Ability:DISORIENTED_AIM_ADJUST/> and Mobility by <Ability:DISORIENTED_MOBILITY_ADJUST/>
+LocPromotionPopupText = "<Bullet/> Works with squadsight. <br/> <Bullet/> <Ability:BLIND_PROTOCOL_COOLDOWN/> turn cooldown. <br/> "
+
+[LW_LeadTheTarget X2AbilityTemplate]
+LocFriendlyName = "Lead The Target"
+LocFlyOverText = "Lead The Target"
+LocHelpText = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown."
+LocLongDescription = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown."
+LocPromotionPopupText = "<Bullet/> Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown <br/>"
```

Comment:
```suggestion
LocHelpText = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemy's turn. Costs two action points and has a <Ability:LEAD_TARGET_COOLDOWN/>-turn cooldown."
LocLongDescription = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemy's turn. Costs two action points and has a <Ability:LEAD_TARGET_COOLDOWN/>-turn cooldown."
LocPromotionPopupText = "<Bullet/> Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemy's turn. Costs two action points and has a <Ability:LEAD_TARGET_COOLDOWN/>-turn cooldown. <br/>"
```

Reply:
When exactly is the shot taken? Right at the start of the enemy's turn, but before any of the enemies get a chance to act? What happens if the Lost take a turn in between this ability being used on an ADVENT unit and the alien turn?